content/conditions/cold/main-content.md

```diff
@@ -58,3 +58,36 @@ There’s little evidence that supplements (such as vitamin c, zinc, echinacea o
 
 GPs don’t usually recommend antibiotics for colds because they won’t relieve your symptoms or speed up your recovery.
 Antibiotics are only effective against bacterial infections and colds are caused by viruses.
+
+!!! alert
+  ## See someone urgently if  
+  
+  - you have difficulty breathing 
+  - you’re coughing up blood 
+  - you have chest pain
+  
+  [Find services that provide urgent care in your area](http://www.nhs.uk/service-search/Minor-injuries-unit/LocationSearch/551)
```

Comment:
Check this link with someone from content as this doesn't feel like the right place to send someone with a cold